main.lua

```diff
@@ -25,13 +25,13 @@ local voice = {} -- list of playing sounds
 -- call this in the sorting code
 local function triggerSound(val, maxval)
   local norm = (val / maxval)                   -- [0,1]
-  local pitch = 220 + 2 ^ (norm * 12 / 12)      -- restrict to one octave, but it does simplify, unless i messed it up
+  local pitch = 440.00 * (2 ^ (norm * 12/12))   -- restrict to one octave, but it does simplify, unless i messed it up
```

Comment:
Is `norm * 12 / 12` needed here?

Reply:
It can be norm as well; the idea is that multiplying a base frequency by 2^(n/12) makes equally spaced semitones across one octave; current implementation basically tries to fit as many individual pitches into said one octave, as there are distinct values - this can be changed, e.g.:
```lua
local pitch = 220.00 + 2 ^ (val / 12)
```

I did just realized that i need to fix 1 more thing though...